internal/plugin/native/manager.go

```diff
@@ -628,7 +628,6 @@ func (rr *Manager) loadRuntime(t plugin2.PluginType, soName, soFilepath, symbolN
 		} else {
 			mod, err := rr.getSoFilePath(t, soName, false)
 			if err != nil {
-				conf.Log.Warnf(fmt.Sprintf("cannot find the native plugin %s in path: %v", soName, err))
```

Comment:
Will this generate a lot of logs especially for function plugins? I suggest to use Debugf here